packages/sdk/src.ts/constants.ts

```diff
@@ -77,3 +77,178 @@ export const SEPOLIA_CUSTOM_ASSET_MAPPINGS = findCustomTokenAddresses(
 export const REVERSE_SEPOLIA_CUSTOM_ASSET_MAPPINGS = flipAndLowerCase(
   SEPOLIA_CUSTOM_ASSET_MAPPINGS
 );
+
+// Network Assets
+export const AVAILABLE_ASSETS: Record<
```

Comment:
did you copy this stuff from the FE? If so did you remove them from the FE so it's not duplicated?

Reply:
I didn't remove them yet. Will remove them once we push a new version of the sdk and update the FE sdk